### PR TITLE
COS-4938 [Dashboard] Validate the launch button for Credit Builder

### PR DESCRIPTION
**Pull request by MiltonDnB**
**[DESCRIPTION OF CHANGES]**
1. List out all the changes being done 
 - Created test to launch CreditBuilder builder from the dashboard.
 - Updated the Launch CreditSignal test user/password

**[EXTRA CONSIDERATIONS WHEN REVIEWING CODE]**
1. Anything else important to consider when reviewing this code?

N/A

**[TESTING STEPS TAKEN]**

1. How was this tested? Screenshots to support?
 - Verified new and existing stories in QA
QA_allstories-COS-4938

 - Verified new and existing stories in STG
STG_allstories-COS-4938

**[ASSOCIATED ISSUE]**

https://github.com/dandb/phoenix/issues/##
https://dunandb.jira.com/browse/COS-4938

### CODE REVIEW

File: regression/src/main/java/com/dandb/ecomm/test/steps/dashboard/LaunchCreditBuilderSteps.java [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+    @Autowired
+    protected DashboardPages pages;
+
+    /**
+     * Checks that the user is entitled to the CreditBuilder product
+     */
+    @Then("I verify CreditBuilder Entitlement")
+    public void validateCreditBuilderTile() {
+        pages.LaunchCreditBuilderPage().validateCreditBuilderTile();
```
Comment (saihareesu) [inline regression/src/main/java/com/dandb/ecomm/test/steps/dashboard/LaunchCreditBuilderSteps.java:20]: Which page is LaunchCreditBuilderPage?

File: regression/src/main/java/com/dandb/ecomm/test/pages/dashboard/LaunchCreditBuilderPage.java [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+public class LaunchCreditBuilderPage extends BasePage {
+
+    private org.slf4j.Logger log = org.slf4j.LoggerFactory.getLogger(getClass());
+
+    public LaunchCreditBuilderPage(WebDriver webDriver) {
+        super(webDriver); }
+
+    @Override
+    public String appUrl() {
+        return appConfig().get(DASHBOARD_APPURL_PROPERTY);
```
Comment (saihareesu) [inline regression/src/main/java/com/dandb/ecomm/test/pages/dashboard/LaunchCreditBuilderPage.java:18]: This is not required as this is already defined in dashboard page.

File: regression/src/main/java/com/dandb/ecomm/test/steps/dashboard/LaunchCreditBuilderSteps.java (hunk 1 above)
Comment (MiltonDnB) [reply regression/src/main/java/com/dandb/ecomm/test/steps/dashboard/LaunchCreditBuilderSteps.java]: Made the correction
CB_Correction

File: regression/src/main/java/com/dandb/ecomm/test/pages/dashboard/LaunchCreditBuilderPage.java (hunk 2 above)
Comment (MiltonDnB) [reply regression/src/main/java/com/dandb/ecomm/test/pages/dashboard/LaunchCreditBuilderPage.java]: Corrections done
remove_appurl

**Comment by MiltonDnB** [comment]
All requested changes made and "AllStories" successfully re-run in QA and STG
QA-STG_allstories-COS-4938

**Pull request merged**